vnext/Microsoft.ReactNative/Views/FrameworkElementViewManager.cpp

```diff
@@ -526,29 +528,79 @@ bool FrameworkElementViewManager::UpdateProperty(
       DynamicAutomationProperties::SetAccessibilityStateCollapsed(
           element, states[static_cast<int32_t>(winrt::Microsoft::ReactNative::AccessibilityStates::Collapsed)]);
     } else if (propertyName == "accessibilityValue") {
+      winrt::hstring textValue;
+      double numericValues[static_cast<int32_t>(
+          magic_enum::enum_count<winrt::Microsoft::ReactNative::AccessibilityValue>() - 1)] = {};
+
       if (propertyValue.Type() == winrt::Microsoft::ReactNative::JSValueType::Object) {
         for (const auto &pair : propertyValue.AsObject()) {
           const std::string &innerName = pair.first;
           const auto &innerValue = pair.second;
 
+          auto peer = xaml::Automation::Peers::FrameworkElementAutomationPeer::FromElement(element);
+
           if (innerName == "min" &&
               (innerValue.Type() == winrt::Microsoft::ReactNative::JSValueType::Double ||
                innerValue.Type() == winrt::Microsoft::ReactNative::JSValueType::Int64)) {
-            DynamicAutomationProperties::SetAccessibilityValueMin(element, innerValue.AsDouble());
+            numericValues[static_cast<int32_t>(winrt::Microsoft::ReactNative::AccessibilityValue::Min)] =
+                innerValue.AsDouble();
+
+            const auto prevMinValue = DynamicAutomationProperties::GetAccessibilityValueMin(element);
+            if (peer != nullptr && prevMinValue != innerValue.AsDouble()) {
+              peer.RaisePropertyChangedEvent(
```

Comment:
Shouldn't these property changed events also get raised if the value isn't set (i.e. it's default is 0.0)? Can this logic be moved into the property setters?

Reply:
do you mean if it goes from being set to not being set? I would imagine if it was intended to be zero, it would be set to zero explicitly. And if it's not set at all, we don't wire through any info about those props to UIA so I don't know if it would make sense for narrator to announce anything.

Reply:
For accessibilityState we do the same thing.